test/cli/other_test.py

```diff
@@ -71,6 +71,68 @@ def test_preprocessor_error(tmpdir):
     assert exitcode != 0
 
 
+ANSI_BOLD = "\x1b[1m"
+ANSI_FG_RED = "\x1b[31m"
+ANSI_FG_DEFAULT = "\x1b[39m"
+ANSI_FG_RESET = "\x1b[0m"
+
+
+def test_clicolor_force(tmpdir):
+    test_file = os.path.join(tmpdir, 'test.c')
+    with open(test_file, 'wt') as f:
+        f.write('#error test\nx=1;\n')
+    exitcode, _, stderr = cppcheck([test_file], env={"CLICOLOR_FORCE":"1"})
+
+    assert exitcode == 0
+    assert ANSI_BOLD in stderr
+    assert ANSI_FG_RED in stderr
+    assert ANSI_FG_DEFAULT in stderr
+    assert ANSI_FG_RESET in stderr
+
+
+@pytest.mark.skipif(sys.platform == "win32", reason="TTY not supported in Windows")
+def test_color_tty(tmpdir):
```

Comment:
refactoring; maybe you could parametrize 

@pytest.mark.parametrize

Reply:
What are you suggesting to parametrize on?

Reply:
I imagine we could parametrize which environment variables are defined. Would it make sense to try both on a tty and on non-tty in Linux?

I don't have a really strong opinion though.